docker-compose.yml

```diff
@@ -20,5 +21,45 @@ services:
     depends_on:
       - todo-service-database
 
+  todo-service-api-2:
```

Comment:
Remove duplication of service. Use `docker compose --sacle` instead

Reply:
Removed duplicates